### PR TITLE
[CONDA] Link LLVM statically

### PR DESCRIPTION
**Pull request by junrushao1994**

### CODE REVIEW

File: conda/recipe/meta.yaml [hunk 1]
```diff
@@ -26,9 +26,9 @@ requirements:
     - {{ compiler('cxx') }} # [not osx]
     - cmake
     - make # [not win]
+    - llvmdev ==10.0.0
   host:
     - zlib
```
Comment (tqchen) [inline conda/recipe/meta.yaml:31]: I believe zlib is also no longer needed and will become a build dep

**Comment by tqchen** [comment]
Looking into the ci error, perhaps we can simply remove zlib as it is implied by llvmdev already

**Comment by junrushao1994** [comment]
@tqchen I checked the log (https://github.com/tlc-pack/tlcpack/pull/35/checks?check_run_id=1787671797), and it looks weird to me that it is examining conflict for every one of the package, just like I was experiencing locally that blocks me to install tlcpack with anaconda. Is that something normal?

**Comment by tqchen** [comment]
I am not too sure, perhaps we can first remove zlib dep and see what happens

**Comment by tqchen** [comment]
Seems due to inavailablity of llvmdev build on this particular platform(ubuntu latest). Perhaps try `llvmdev=10.0.1`?

**Comment by junrushao1994** [comment]
It looks weird that `libgcc-ng` and `libstdcxx-ng` conflict with other packages...

**Comment by junrushao1994** [comment]
Any ideas why it continues to fail...?

**Comment by junrushao1994** [comment]
I can actually build in my local machine...Will try the docker container later

**Pull request closed without merging**